it_fest/models.py

```diff
@@ -3,3 +3,24 @@
 from django.db import models
 
 # Create your models here.
+class Registration(models.Model):
+	GAME_CHOICE = (('Call Of Duty 4','Call Of Duty 4'),
+		('Fifa 17', 'Fifa 17'),
+		('NFS Most Wanted', 'NFS Most Wanted'),)
+	game_name = models.CharField(max_length=15, choices=GAME_CHOICE)
+	transaction_id = models.BigIntegerField()
+	bkash_mobile_number = models.CharField(max_length=14)
```

Comment:
There should be another field with status with choices ``pending`` and ``verified``